python/servo/build_commands.py

```diff
@@ -425,6 +426,15 @@ def build_geckolib(self, with_gecko=None, jobs=None, verbose=False, release=Fals
 
         print("GeckoLib build completed in %s" % format_duration(elapsed))
 
+        if with_gecko is not None and ret == 0:
+            print("Copying binding files to style/gecko_bindings...")
+            build_path = path.join(geckolib_build_path, "release" if release else "debug", "build")
+            target_style_path = find_dep_path_newest("style", build_path)
+            out_gecko_path = path.join(target_style_path, "out", "gecko")
```

Comment:
We don't know whether this path exists. We only know that the `target_style_path` exists. There are cases when we'll be having `build-script-build` file instead of the `out`directory, when this will raise an exception. In #14601, I have an optional check which fixes this. Once we find the path, we should make sure that `target_style_path is not None` [like we do elsewhere](https://github.com/servo/servo/blob/d03039757c3fc9b86152481a5dc4a54a0ecc18e4/python/servo/package_commands.py#L137-L138).